applications/openshift/general/acs_sensor_exists/tests/ocp4/e2e.yml

```diff
@@ -0,0 +1,2 @@
+---
+default_result: FAIL
```

Comment:
This goes back to @yuumasato's comment, but now that we have a manual remediation, we should be testing that it works in the second scan, right?